syncopy/datatype/base_data.py

```diff
@@ -738,9 +738,22 @@ def __del__(self):
                 shutil.rmtree(os.path.splitext(self.filename)[0],
                               ignore_errors=True)
 
+    # Support for basic arithmetic operations (no in-place computations supported yet)
     def __add__(self, other):
         return _process_operator(self, other, "+")
 
+    def __sub__(self, other):
+        return _process_operator(self, other, "-")
```

Comment:
Overloading with non-member functions probably isn't standard, yet here absolutely the right way.. also looks super clean :+1:

Reply:
:D Thanks. Necessity and invention and all that